gunicorn_thrift/thriftpy_gevent_worker.py

```diff
@@ -31,6 +37,23 @@
 logger = logging.getLogger(__name__)
 
 
+# Take references to un-monkey-patched versions of stuff we need.
+# Monkey-patching will have already been done by the time we come to
+# use these functions at runtime.
+# And there's a change thread is already patched when importing this module,
+# so we check first if it's already patched.
+def __get_real_function(module, name):
```

Comment:
We will patch `thread` before forking, so when importing this module, we have to get the original from gevent.monkey